### PR TITLE
OperatoinsCache service

### PR DESCRIPTION
**Pull request by samodovdi**

### CODE REVIEW

File: src/Lykke.Service.OperationsCache/Modules/ServiceModule.cs [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+            builder.RegisterType<ShutdownManager>()
+                .As<IShutdownManager>();
+
+            builder.RegisterType<HistoryCache>()
+                .WithParameter(TypedParameter.From(_settings.CurrentValue.OperationsCacheService.ItemsPerPage))
+                .As<IHistoryCache>()
+                .SingleInstance();
+
+            builder.Populate(_services);
```
Comment (NovichikhinAlexey) [inline src/Lykke.Service.OperationsCache/Modules/ServiceModule.cs:53]: Populate after AddDistributedRedisCache ... please check that redis really work
https://lykkex.atlassian.net/wiki/spaces/LKEWALLET/pages/177111113/Working+with+Redis
"Oh, seems like it works in-memory"

File: src/Lykke.Service.OperationsCache/Settings/ServiceSettings/OperationsCacheSettings.cs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+﻿namespace Lykke.Service.OperationsCache.Settings.ServiceSettings
+{
+    public class OperationsCacheSettings
+    {
+        public string CacheInstanceName { get; set; }
```
Comment (NovichikhinAlexey) [inline src/Lykke.Service.OperationsCache/Settings/ServiceSettings/OperationsCacheSettings.cs:5]: Usually, in this section the Radis section is registered and there is a link to the access point and the name of the instance

Not understand for what we add separte section RedisSettings

File: src/Lykke.Service.OperationsCache/Modules/ServiceModule.cs (hunk 1 above)
Comment (samodovdi) [reply src/Lykke.Service.OperationsCache/Modules/ServiceModule.cs:53]: no, its ok here, i've created it in the constructor. Populate is done in Startup.cs before the module initializations, so its ok and redis cache is used

File: src/Lykke.Service.OperationsCache/Settings/ServiceSettings/OperationsCacheSettings.cs (hunk 2 above)
Comment (samodovdi) [reply src/Lykke.Service.OperationsCache/Settings/ServiceSettings/OperationsCacheSettings.cs:5]: just made the same as for the OperationsCache job

**Review by NovichikhinAlexey — APPROVED** [review]

**Pull request merged**